wagtail/admin/templates/wagtailadmin/pages/editor_sidebar_panels/includes/page_privacy_status.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'wagtailadmin/pages/editor_sidebar_panels/includes/action_list_item.html' %}
+{% load i18n wagtailadmin_tags %}
+
+{% block content %}
+    {% test_page_is_public page as is_public %}
+
+    {# The swap between public and private text is done using JS inside of privacy-switch.js when the response from the modal comes back #}
+    <div class="{% if not is_public %}w-hidden{% endif %}" data-privacy-sidebar-public>
+        {% trans 'Visible to all' as title %}
+        {% trans 'Once live anyone can view' as help_text %}
+        {% with icon_name='view' %}
+            {{ block.super }}
+        {% endwith %}
+    </div>
+    <div class="{% if is_public %}w-hidden{% endif %}" data-privacy-sidebar-private>
+        {% trans 'Private' as title %}
+        {% trans 'Not visible to the public' as help_text %}
+        {% with icon_name='no-view' %}
+            {{ block.super }}
+        {% endwith %}
+    </div>
+{% endblock %}
+
+{% block action %}
+    {% page_permissions page as page_perms %}
+    {% if page_perms.can_set_view_restrictions %}
+        {% trans 'Set privacy' as set_privacy_text %}
+        <button class="action-set-privacy w-bg-transparent w-text-14 w-p-0 w-text-teal-200 hover:w-underline" data-url="{% url 'wagtailadmin_pages:set_privacy' page.id %}" type="button">{{ set_privacy_text }}</button>
```

Comment:
I think it would be nice to either make a component for this button style, or a CSS class, so we don’t repeat it across so many files and risk inconsistencies cropping up.